scylla-cql/Cargo.toml

```diff
@@ -14,29 +14,52 @@ license = "MIT OR Apache-2.0"
 all-features = true
 
 [dependencies]
+###########################
+# Main, public dependencies
+###########################
 # Important: We use precise version of scylla-macros. This enables
 # us to make breaking changes in the doc(hidden) interfaces that are
 # used by macros.
 scylla-macros = { version = "=1.2.0", path = "../scylla-macros" }
-byteorder = "1.3.4"
-bytes = "1.0.1"
+# AsyncRead trait used in read_response_frame
 tokio = { version = "1.40", features = ["io-util", "time"] }
-secrecy-08 = { package = "secrecy", version = "0.8", optional = true }
-snap = "1.0"
+# FrameSlice and other parts of public API
+bytes = "1.0.1"
+# Tracing ids, CqlTimeuuid, ser/deser of CQL UUID
 uuid = "1.0"
-thiserror = "2.0.6"
-itertools = "0.14.0"
-num-bigint-03 = { package = "num-bigint", version = "0.3", optional = true }
-num-bigint-04 = { package = "num-bigint", version = "0.4", optional = true }
-bigdecimal-04 = { package = "bigdecimal", version = "0.4", optional = true }
+
+#################################
+# Serialization / deserialization
+#################################
+# Ser/deser of time-related types. Not optional, because it is also used by Display impl
+# of CqlValue
 chrono-04 = { package = "chrono", version = "0.4.32", default-features = false, features = [
     "alloc",
 ] }
-lz4_flex = { version = "0.11.1" }
+secrecy-08 = { package = "secrecy", version = "0.8", optional = true }
+num-bigint-03 = { package = "num-bigint", version = "0.3", optional = true }
+num-bigint-04 = { package = "num-bigint", version = "0.4", optional = true }
+bigdecimal-04 = { package = "bigdecimal", version = "0.4", optional = true }
 serde = { version = "1.0", features = ["derive"], optional = true }
```

Comment:
:question: What is serde used for?

Reply:
Huh, I missed that one.

It is used for ser/deser of Consistency and SerialConsistency. This in turn is used by `unstable-cloud` in `scylla`.
I'll add description to it.